LS_Miroshnichenko/main.swift

```diff
@@ -8,5 +8,35 @@
 
 import Foundation
 
-print("Hello, World!")
+let clHelper = CommandLineIOHelper()
+var userArguments = clHelper.input()
 
+if userArguments.count < 2 {
+    clHelper.output(outputText: "parameter is not defined, open help with parameter \"-h\"", outputType: .error)
+    exit(0)
+}
+
+if userArguments.contains("-h") {
+    clHelper.help()
+    exit(0)
+}
+
+let fileManager = FileManager.default
+let currenDirectoryPath = fileManager.currentDirectoryPath
+let shouldSkipHiddenFiles = !userArguments.contains("-a")
+
+guard let urls = fileManager.urls(for: currenDirectoryPath, skipsHiddenFiles: shouldSkipHiddenFiles) else {
+    exit(0)
+}
+
+let shouldPrintAttributes = userArguments.contains("-l")
```

Comment:
I'd suggest to create an enum of possible attributes and map arguments to an array of this enum cases. 
It is not a big win in this case but assuming we need to reuse this attributes somewhere else in future it's better to have consistent enum cases for them instead of "magic strings"